frontends/concrete-python/examples/game_of_life/game_of_life.py

```diff
@@ -1,16 +1,18 @@
+import math
```

Comment:
one possible convention
first by alpha order things from the stdlib
second by alpha order things from external libs
then our internal stuff (not present in this case)

```python
import argparse
...
import time

environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "1"

import numpy as np
from concrete import fhe
import pygame
```

Reply:
should be good now